scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -172,6 +199,9 @@ trait Config {
   def getUniqueId: Option[UniqueID] =
     get(UniqueID.UNIQUE_JOB_ID).map(UniqueID(_))
 
+  def addCascadingClassSerializationTokens(clazzes: Set[Class[_]]): Config =
```

Comment:
Let's add a comment here to the effect of:

The serialization of your data will be smaller if any classes passed between tasks in your job are listed here. Without this, strings are used to write the types IN EACH RECORD, which compression probably takes care of, but compression acts AFTER the data is serialized into buffers and spilling has been triggered.